mmv1/products/artifactregistry/Repository.yaml

```diff
@@ -369,10 +379,22 @@ properties:
                   - :UBUNTU
               - !ruby/object:Api::Type::String
                 name: 'repositoryPath'
-                required: true
```

Comment:
Removing these `required` tags since these shouldn't be set if a custom_repository is chosen, and I believe keeping these will cause issues (I originally added `required` on the custom_repository uri field and this caused issues with public_repository tests). But if we can keep these `required` fields to only show up when `public_repository` is used and not `custom_repository`, I can re-add these!